Lib/lib2to3/pgen2/tokenize.py

```diff
@@ -142,7 +149,9 @@ def maybe(*choices): return group(*choices) + '?'
           "ur'", 'ur"', "Ur'", 'Ur"',
           "uR'", 'uR"', "UR'", 'UR"',
           "br'", 'br"', "Br'", 'Br"',
-          "bR'", 'bR"', "BR'", 'BR"', ):
+          "bR'", 'bR"', "BR'", 'BR"',
+          "rb'", 'rb"', "Rb'", 'Rb"',
+          "rB'", 'rB"', "RB'", 'RB"',):
```

Comment:
I agree this is not elegant. Sadly, the list is not exhaustive as the `_litprefix` regex shows. So the algorithm would need to be custom anyway. I'd count this as a possible future improvement.